DiscordBotApis/Data/MockGuildRepository.cs

```diff
@@ -12,19 +12,19 @@ public List<Guild> GetGuilds()
         {
             var guild = new Guild()
             {
-                Name = "Bongus"
+                //Name = "Bongus"
```

Comment:
I think it's okay to comment this out until we update the mock stuff.

Reply:
Fair enough... took it out anyways so it looks a little cleaner.